common/src/main/java/io/netty/util/Recycler.java

```diff
@@ -89,8 +89,8 @@ public void recycle(Object object) {
 
         if (logger.isDebugEnabled()) {
             if (DEFAULT_MAX_CAPACITY_PER_THREAD == 0) {
-                logger.debug("-Dio.netty.recycler.maxCapacityPerThread: disabled");
-                logger.debug("-Dio.netty.recycler.maxSharedCapacityFactor: disabled");
+                logger.debug("-Dsiren.io.netty.recycler.maxCapacityPerThread: disabled");
+                logger.debug("-Dsiren.io.netty.recycler.maxSharedCapacityFactor: disabled");
```

Comment:
It seems to be an pre-existing error, but that property shouldn't prefixed with `siren`: on line 74 https://github.com/sirensolutions/netty/pull/17/files?file-filters%5B%5D=.java&file-filters%5B%5D=.xml#diff-c4c0f035e26b5f8983f985f3e3368a0dfc8fc75e982268e53150af81da1d94acR74 we don't get the siren's `maxSharedCapacityFactor`.

Reply:
thanks done https://github.com/sirensolutions/netty/pull/17/commits/9c7efd481a031ffd2ef635ad8c0f95e913516b84